pydantic/_internal/_serializers.py

```diff
@@ -1,31 +1,12 @@
 from __future__ import annotations as _annotations
 
-from collections import deque
 from typing import Any, Pattern
 
-from pydantic_core import PydanticOmit
-from pydantic_core.core_schema import SerializationInfo, SerializerFunctionWrapHandler
+from pydantic_core.core_schema import SerializationInfo
 
 
 def pattern_serializer(input_value: Pattern[Any], info: SerializationInfo) -> str | Pattern[Any]:
     if info.mode == 'json':
         return input_value.pattern
     else:
         return input_value
-
-
-def serialize_deque(
```

Comment:
This function is not in use after https://github.com/pydantic/pydantic/commit/3c020df9ffa87c33f9d6c886bb5527c1feb8b39a